lib/charms/mongodb/v1/mongodb.py

```diff
@@ -124,50 +103,9 @@ def __init__(self, config: MongoDBConfiguration, uri=None, direct=False):
             direct: force a direct connection to a specific host, avoiding
                     reading replica set configuration and reconnection.
         """
+        MongoConnection.__init__(self, config, uri, direct)
```

Comment:
Use `super()` here like so:
```python
super().__init__(self, config, uri, direct)
```
Also, considering that we save the config here, I'm wondering if there's a reason not to save it directly in the base class?

Reply:
🧠